newrelic/data_source_newrelic_entity.go

```diff
@@ -144,7 +150,13 @@ func dataSourceNewRelicEntityRead(ctx context.Context, d *schema.ResourceData, m
 	}
 
 	if entity == nil {
-		return diag.FromErr(fmt.Errorf("no entities found for the provided search parameters, please ensure your schema attributes are valid"))
+		if d.Get("ignore_not_found").(bool) {
+			log.Printf("[INFO] Entity not found, ignoring error")
+			d.SetId("")
+			return nil
```

Comment:
```suggestion
			var diags diag.Diagnostics
			diags = append(diags, diag.Diagnostic{
				Severity: diag.Warning,
				Summary:  "no entities found for the provided search parameters, please ensure your schema attributes are valid",
			})
			return diags
```

(please **do not** commit this already - this is a suggestion I have, in order to return a warning instead of an error when an entity is not found and the `ignore_not_found` attribute is used, to avoid halting execution caused by an error; but I'd like more in my team to take a look at this and judge if this change makes sense. Cheers!)

Reply:
ok, that makes sense. thank you